app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java

```diff
@@ -254,14 +269,14 @@ private void pause() {
      */
     private void setPaused() {
         pauseResumeButton.setImageResource(R.drawable.pause_icon);
-        pauseResumeButton.setTag(R.string.pause);
+        pauseResumeButton.setTag("pause");
```

Comment:
Hey, if we just pass the string by it's id in setTag, it will set the id of resource like 215678942, like that, not the value of the resource, that's why i am passing like this

Reply:
And I do this to solve the issue no. #4840

Reply:
@nicolas-raoul, should I change this ?

Reply:
changing the text to resource create an issue

Reply:
@4D17Y4 By any chance, would you have any advice on this?

Reply:
thanks for updating the values @arinmodi

Reply:
@4D17Y4, is this okay now ?